InstantPrintTool.py

```diff
@@ -83,7 +117,14 @@ def setEnabled(self, enabled):
     def __changeScale(self):
         if not self.mapitem:
             return
-        newscale = self.dialogui.spinBoxScale.value()
+        try:
+            newscale = 1. / self.dialogui.comboBox_scale.scale()
+        except ZeroDivisionError:
+            return
+        # if 0 is divided by 0 the Exception doesn't catch it
+        # So I to catch it like this
+        if not newscale:
+            return
```

Comment:
When is zero divided by zero here, given that the nominator is 1 in the operation?